R/segmentPattern.R

```diff
@@ -229,8 +216,7 @@ segmentPattern <- function(x,
                            run.parallel = FALSE,
                            run.parallel.cores = 1L,
                            x.cut = TRUE,
-                           x.cut.vl = 6000,
-                           compute.template.idx = FALSE){
```

Comment:
Thanks, will handle this last once we settle on which optimizations we're doing and how that affects the interface, just in case we end up messing with some of the arguments further.